### PR TITLE
settingUp.md: add instructions on how to log in as a student #1745

### LINKED ISSUE #1745: SettingUp.md: add instructions on how to login as a student
Here is the outline:

First, login as an instructor and add some students.
Then, login as admin and search for the student. The search results will give you the 'join link' for that student. Logout from the instructor account and use that 'join link' to login. Remember to use 'http://localhost:8888' as the host instead of the one given in the join link.
That should allow you to access dev server as a student.

Alternatively, run the test cases first. That creates several student accounts in the datastore. Use one of them to login as a student.

### PR DESCRIPTION
**Pull request by drunken-saint**
Fixes #1745

### CODE REVIEW

**Comment by kanghj** [comment]
Looks good to merge

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -75,7 +75,12 @@ Important: When a version is specified, please install that version instead of t
    Instead, you can use the join link given after adding an intructor, to complete the joining process.<br>
    Remember to change the URL of the link if necessary.<br>
    e.g. Change `http://teammates-john.appspot.com/page/instructorCourseJoin?...` to `http://localhost:8888/page/instructorCourseJoin?...`
-      
+8. Now, to access the dev server as a student, first make sure you are logged in as an instructor. Add a course for yourself and then add the students for the course.<br>
+   After that, log in as admin and search for the student you added in as instructor. From the search results, use the join link adjacent to the student you added.<br>
+   Then, log out and use that join link to log in as a student. You have the required access now.<br>
+   (Make sure you use the `http://localhost:8888/` as the host instead of the one given in the join link)<br>   
+   Alternative : Run the test case, the test case creates several student account in the datastore. Use one of them to login.<br>
```
Comment (damithc) [inline devdocs/settingUp.md]: test cases?

**Comment by damithc** [comment]
Minor change requested. 
@drunken-saint please create a separate issue about the 'login' --> 'log in' problem you noticed.

**Comment by drunken-saint** [comment]
So, what phrase should I use in this fix ? "login" or "log in" ?

**Comment by damithc** [comment]
log in for verb e.g. log in to the website
login for noun e.g. the login page

**Comment by drunken-saint** [comment]
@damithc  Sir , I've already discussed the test cases in the contributors group with @kanghj . 
I'm attaching the link too. I think, it all looks good to be merged now .
https://groups.google.com/forum/#!topic/teammates-contributors/FE-Slm0-GbU

Thank you.

File: devdocs/settingUp.md [hunk 2]
```diff
@@ -75,7 +75,12 @@ Important: When a version is specified, please install that version instead of t
    Instead, you can use the join link given after adding an intructor, to complete the joining process.<br>
    Remember to change the URL of the link if necessary.<br>
    e.g. Change `http://teammates-john.appspot.com/page/instructorCourseJoin?...` to `http://localhost:8888/page/instructorCourseJoin?...`
-      
+8. Now, to access the dev server as a student, first make sure you are logged in as an instructor. Add a course for yourself and then add the students for the course.<br>
+   After that, log in as admin and search for the student you added in as instructor. From the search results, use the join link adjacent to the student you added.<br>
+   Then, log out and use that join link to log in as a student. You have the required access now.<br>
+   (Make sure you use the `http://localhost:8888/` as the host instead of the one given in the join link)<br>   
+   Alternative : Run the test case, the test case creates several student account in the datastore. Use one of them to log in.<br>
```
Comment (damithc) [inline devdocs/settingUp.md]: The line above says 'test case'. That should be 'test cases' (plural)

**Comment by drunken-saint** [comment]
Corrected that too. Ran the tests again . Only a single test failed (InstructorsFeedbackResultsPageUiTest).It is the one which @kanghj  told me to ignore as it fails in some cases .

**Comment by damithc** [comment]
go ahead and merge @kanghj

**Pull request merged**